.github/workflows/ci-quarkus-container-scan.yml

```diff
@@ -0,0 +1,94 @@
+name: Container scan
+
+on:
+  workflow_call:
+    inputs:
+      image-name:
+        description: Name of Docker image
+        required: false
+        type: string
+      image-pack:
+        description: Docker image pack of builder-jammy-tiny, builder-jammy-base or builder-jammy-full
+        default: builder-jammy-tiny
+        required: false
+        type: string
+      registry-url:
+        description: Image/Container Registery URL
+        default: 'my-local-registery'
+        required: false
+        type: string
+      java-version:
+        description: Main version of java
+        default: '11'
+        required: false
+        type: string
+      container-scan-offline-mode:
+        description: Container-scan is default download updated CVE definitions via Trivy, enable offline on download problems
+        default: false
+        type: boolean
+
+jobs:
+  build-and-scan-image:
+      runs-on: ubuntu-latest
+      env:
+        REPOSITORY-NAME: ${{ github.event.repository.name }}
+        DOCKLE_HOST: "unix:///var/run/docker.sock"
+        #TRIVY_TIMEOUT: "15m"
+        TRIVY_OFFLINE_SCAN: ${{ inputs.container-scan-offline-mode }}
+      steps:
+        - name: Set imagetag as env variable
+          run: echo "IMAGETAG=$(date +'%Y-%m-%d-%H%M')-${GITHUB_SHA::8}" >> "$GITHUB_ENV"
+        - name: Set IMAGE-NAME env variable
+          run: |
+             echo "IMAGE-NAME=${{ inputs.registry-url }}/${{ inputs.image-name || env.REPOSITORY-NAME }}" >> "$GITHUB_ENV"
+        - uses: actions/checkout@v4
+        - name: Cache Maven packages
+          uses: actions/cache@v4.0.0
+          with:
+            path: ~/.m2
+            key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+            restore-keys: ${{ runner.os }}-m2
+        - uses: s4u/maven-settings-action@v3.0.0
+          with:
+            servers: |
+              [{
+                  "id": "github-oidc-sdk",
+                  "username": "${{ secrets.MAVEN_USER }}",
+                  "password": "${{ secrets.MAVEN_PASSWORD }}"
+              },
+              {
+                "id": "github",
+                "username": "${{ secrets.MAVEN_USER }}",
+                "password": "${{ secrets.MAVEN_PASSWORD }}"
+              }]
+        - name: Install pack
+          run: |
+            #!/usr/bin/env bash
+            set -euo pipefail
+            echo "Installing pack ${PACK_VERSION}"
+            mkdir -p /usr/local/bin
+            curl \
+              --location \
+              --show-error \
+              --silent \
+              "https://github.com/buildpacks/pack/releases/download/v${PACK_VERSION}/pack-v${PACK_VERSION}-linux.tgz" \
+            | sudo tar -C /usr/local/bin/bin --no-same-owner -xzv pack
+          env:
+            PACK_VERSION: 0.32.1
```

Comment:
kunne ein berre brukt mvn ala https://quarkus.io/guides/maven-tooling#build-a-container-friendly-executable eller blir det ikkje "distroless" image då kanskje? blir stor?  litt på tynn is akkurat her, men ser ut som quarkus maven plugin har det ein treng av settings elles som registery/version mm.

Reply:
Eg berre antok at de ville at me skulle bruke paketo, for å få det så likt som mogleg som dei andre... elles er det godt mogleg å bruke quarkus sine image tools.. det er fleire https://quarkus.io/guides/container-image#buildpack